config/lint/lint.xml

```diff
@@ -22,6 +22,18 @@
         <ignore path="**/generated/source/kapt/**/org/wordpress/android/push/Hilt_GCMMessageService.java" />
     </issue>
     <issue id="GradleDependency" severity="ignore" /> <!-- Dependabot will take care of this -->
+    <issue id="MissingNullAnnotationOnField">
+        <ignore path="**/generated/**" />
+    </issue>
+    <issue id="MissingNullAnnotationOnConstructorParameter">
+        <ignore path="**/generated/**" />
+    </issue>
+    <issue id="MissingNullAnnotationOnMethodParameter">
+        <ignore path="**/generated/**" />
+    </issue>
+    <issue id="MissingNullAnnotationOnMethodReturnType">
+        <ignore path="**/generated/**" />
+    </issue>
```

Comment:
Would it be possible to add this config as a part of `wordPressLintVersion`? I think that we never really want to lint `generated` code in any way.

Reply:
Good question @wzieba ! 💯 

**TL:DR**: We had a looooong discussion about that and ended-up merging this Lint rule without it, you could check [this](https://github.com/wordpress-mobile/WordPress-Lint-Android/pull/11#issuecomment-1689389557) last comment on it, and all the relevant comments further up, I think you'll get the idea and the why behind this decision.

Let me know if that makes sense to you too. 😄